.docker/docker-entrypoint-initdb.d/init.txt

```diff
@@ -170,6 +170,7 @@ CREATE TABLE event (
   index             int NOT NULL CHECK (index >= 0),
   id                bigserial NOT NULL PRIMARY KEY,
   data              bytea NULL CHECK (length(data) > 0),
+  "from"            address NOT NULL,
```

Comment:
why does `from` have to be in `"`?

Reply:
Because `from` is a reserved sql keyword.

Reply:
makes sense